packages/js/components/src/split-dropdown/test/index.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { createElement } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { SplitDropdown } from '..';
+
+describe( 'SplitDropdown', () => {
+	test( 'it renders correctly collapsed', () => {
```

Comment:
Very simple tests, with no interaction. Let me know if you think there's something we have to add.